packages/eds-lab-react/src/components/SideBar/SideBar.docs.mdx

```diff
@@ -25,28 +25,70 @@ It has a toggleable collapsed and expanded state.
 
 <ul>
     <li>The Sidebar does not scroll away, being a consistent guidepost to users.</li>
+    <li><code>SideBar.Content</code> and <code>SideBar.Footer</code> are subcomponents for layout. 
+    You should normally use these to contain the other content.</li>
+    <li><code>SideBar.Toggle</code> is optional and should be placed either at the top or in the footer.</li>
 </ul>
 
 ```tsx
-import { SideBar } from '@equinor/eds-lab-react'
-
-<SideBar>
-    <SideBar.Content>
-      <SideBar.Toggle />
-      <SideBar.Link name="home" icon={home} href="/" />
-      <SideBar.Link name="dashboard" icon={dashboard} href="/dashboard" />
-    </SideBar.Content>
-</SideBar>
+import { SideBar, SidebarLinkProps } from '@equinor/eds-lab-react'
+  const menuItemsInitial: SidebarLinkProps[] = [
+    {
+      name: 'Home',
+      icon: home,
+      href: '/',
+      active: false,
+    },
+    {
+      name: 'Dashboard',
+      icon: dashboard,
+      href: '/dashboard',
+      active: false,
+    },
+  ]
+  const [menuItems, setMenuItems] = useState(menuItemsInitial)
+  const updateActive = () => {
+    /* Logic to update active state of links */
+  }
+
+  <SideBar>
+      <SideBar.Content>
+        <SideBar.Toggle />
+        {menuItems.map((m) => (
+            <SideBar.Link key={m.name} {...m} />
+          ))}
+      </SideBar.Content>
+  </SideBar>
 ```
 
 ## Examples
 
-### With custom content
+### With TopBar
+If used together with `TopBar`, the layout should be handled with css-grid to ensure the sidebar does not extend beyond the screen and creates overflow.
+<Story id="components-sidebar--with-topbar" />
+
+### SideBar.Link
 
-Use <code>useSideBar</code> to detect sidebar state in your custom content if it needs to be modified based on open/closed state.
+`SideBar.Link` requires both `label` and `icon` due to the visual style of the collapsed state. 
+Navigation can be done with `href`, or in the case of a third party routing library, one can use the `as` prop 
+to override component type and allow their specific props. 
+`SideBar.Link` have an `active` prop that should be used to visualize the current path. 
+We do not provide logic for detecting current path, as different projects and third party routers 
+have different needs and behaviours around this, and may have their own tools or hooks to simplify this.
```

Comment:
Maybe something shorter? This ended up as big block of text and different needs for project goes for a lot of other features of our components so its kinda of a given at this point.

```suggestion
Use the `active` property to indicate current active path when used for navigation.
```